static/app/views/automations/components/automationBuilder.tsx

```diff
@@ -142,11 +142,19 @@ function ActionFilterBlock({actionFilter}: ActionFilterBlockProps) {
 
     // Only send test notification if there are no validation errors
     if (Object.keys(actionErrors).length === 0) {
-      await sendTestNotification(
-        actionFilterActions.map(action => {
-          return stripActionFields(action);
-        })
-      );
+      try {
+        await sendTestNotification(
+          actionFilterActions.map(action => {
+            return stripActionFields(action);
+          })
+        );
+      } catch (error: any) {
```

Comment:
Instead of catching, the more common usage is to send `onError` to `useSendTestNotification()`. The hook can supply the expected error type so you don't need to type `any` here.

A couple other things: you don't need `mutateAsync` here, you can just use `mutate` this way. Oh, and I think we should be using `isPending` from `useSendTestNotification` to disable the button while the request is in flight.